pkg/services/postgresql/common_bind.go

```diff
@@ -110,6 +110,24 @@ func createCredential(
 		port,
 		databaseName,
 	)
+
+	var jdbcTemplate string
+	if sslRequired {
+		jdbcTemplate = 
+			"jdbc:postgresql://%s:%d/%s?user=%s&password=%s&sslmode=require"
+	} else {
+		jdbcTemplate = "jdbc:postgresql://%s:%d/%s?user=%s&password=%s"
+	}
+
+	jdbc := fmt.Sprintf(
+		jdbcTemplate,
+		fqdn,
+		port,
+		databaseName,
+		url.QueryEscape(username),
+		string(bindDetails.Password),
```

Comment:
It should also use `url.QueryEscape`, right?

Reply:
Quite possibly, I based this on the existing logic - https://github.com/Azure/open-service-broker-azure/blob/master/pkg/services/postgresql/common_bind.go#L108

Will give it a test and see if there's a reason it's not already QueryEscape'd

Reply:
I think it works well because the generated password is alphanumeric (ref: pkg/generate/password.go). But logically it still should use `url.QueryEscape`. It would be better if you'd like to fix both :). Thanks

Reply:
Ah right, that makes sense, hadn't dived that deep I'm afraid!

Reply:
Thanks for fixing it! I am going to merge it pending CI.